### PR TITLE
feat: add variable for aliyun theme of dropdown

### PR DESCRIPTION
**Pull request by hi-zhaoyeqing**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (add variable for aliyun theme of dropdown)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |feat: add variable for aliyun theme of dropdown|
| 🇨🇳 Chinese |新功能：为dropdown组件添加阿里云主题变量|

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by vthinkxie — APPROVED** [review]

File: components/dropdown/style/index.less [hunk 1]
```diff
@@ -216,7 +216,7 @@
 
       &-selected {
         color: @dropdown-selected-color;
-        background-color: @item-active-bg;
+        background-color: @dropdown-selected-bg;
```
Comment (afc163) [inline components/dropdown/style/index.less:219]: 单独提变量的原因是？Dropdown 的选中效果和 Select 不一致么？

Comment (qdzhaoxiaodao) [reply components/dropdown/style/index.less:219]: dropdown阿里云主题目前使用background: rgba(14, 146, 255, 0.05);跟@item-active-bg 不一样,然后就想通过变量更改
image

**Comment by qdzhaoxiaodao** [comment]
本次提交的pr是为了对齐xconsle阿里云主题的dropdown新增的变量，可以控制一级和二级的selected选中背景
image

File: components/dropdown/style/index.less (hunk 1 above)
Comment (afc163) [reply components/dropdown/style/index.less:219]: 阿里云主题的 Select 选中效果和 Dropdown 选中样式不一样是么？

Comment (qdzhaoxiaodao) [reply components/dropdown/style/index.less:219]: 一样，但是select有变量@select-item-selected-bg 可以更改，dropDown没有

**Review by afc163 — APPROVED** [review]

**Review by zombieJ — APPROVED** [review]

**Pull request merged**